lightning/src/ln/channelmanager.rs

```diff
@@ -1339,6 +1342,8 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 			our_network_pubkey: PublicKey::from_secret_key(&secp_ctx, &keys_manager.get_node_secret()),
 			secp_ctx,
 
+			inbound_payments_key: SecretKey::from_slice(&keys_manager.get_secure_random_bytes()).expect("broken RNG"),
```

Comment:
This should be a new method on the `KeysInterface` - users may want to use the same secret across multiple `ChannelManager`s in some cases. I believe with that change the fuzzer will pass as well.